package/keywriter/package

```diff
@@ -5,24 +5,24 @@
 pkgnames=(keywriter)
 pkgdesc="Markdown-enabled free writing app"
 url=https://github.com/dps/remarkable-keywriter
-pkgver=0.1.0-3
-timestamp=2019-07-13T06:27Z
+pkgver=0.2~20-gddc9e73-1
```

Comment:
```suggestion
pkgver=0.2~20-ddc9e732-1
```

I think the commit sha1 short has a type here.

Reply:
I’ll admit it’s a bit strange, but this is the format generated by running `git describe --tags` in the keywriter repo. Not sure why git adds the `g` prefix here.

Reply:
Ah, cool, didn't even know this was [a thing](https://stackoverflow.com/questions/23939214/why-does-git-describe-prefix-the-commit-id-with-the-letter-g). Learned something new I guess.